### ISSUE TITLE
Release v1.0.4

### ISSUE THREAD

**Issue by nihaals** [HandyHat/ha-hildebrandglow-dcc#351]
#331 fixes a fairly large bug but isn't included in the latest release which means most users are unaware of it. Releasing a new version would mean more people would get to use the update as it would show as an update in HACS.

**Comment by cannfoddr** [comment]
Agreed - I've forked the repository as a test and done a quick and dirty merge of all the outstanding PRs.  Had to do a bit of cleanup.  I then dropped the custom integration folder over the top of my current HACS install.

So far so good I have entities for GAS and ELEC with Values but for some reason I am only seeing ELEC in the energy dashboard.

**Comment by nihaals** [comment]
I installed `main` which has been working well, although I only recently discovered that the integration was fixed

**Comment by nihaals** [comment]
This installs the latest commit on `main`:
<img width="564" alt="image" src="https://github.com/HandyHat/ha-hildebrandglow-dcc/assets/18350092/50ceb108-222e-4831-a567-1c87ac92d807">

**Comment by leaskovski** [comment]
Thanks.. I thought there was a way to do it via the UI, but couldn't remember, and then stumbled across it just before you posted this, so thanks!  Still hasn't sorted my gas readings though :(

**Comment by RobertClarke64** [comment]
@HandyHat could we get an update on this? I think a new release tag would really help users, especially since the merge of #331.

**Comment by sebastianovide** [comment]
I don't think this is longer supported.. look at the dates...
